components/org.wso2.carbon.identity.recovery/src/main/java/org/wso2/carbon/identity/recovery/util/Utils.java

```diff
@@ -427,6 +427,19 @@ public static String doHash(String value) throws UserStoreException {
         }
     }
 
+    /**
+     * @param value     Value to be hashed
+     * @return          Hashed value
+     * @throws NoSuchAlgorithmException If the algorithm is not found.
+     */
+    public static String hashCode(String value) throws NoSuchAlgorithmException {
```

Comment:
You can use this method call inside `doHash` method and deprecate it.

Reply:
Fixed with f742d185c13617f54d6f9213f154c2acad87ac3e